NeonShooter/Game.cs

```diff
@@ -42,10 +42,9 @@ public NeonShooterGame()
         {
 			Instance = this;
 			graphics = new GraphicsDeviceManager(this);
-			Content.RootDirectory = "Content";
 
-            graphics.PreferredBackBufferWidth = 1920;
-            graphics.PreferredBackBufferHeight = 1080;
+            //graphics.PreferredBackBufferWidth = 1920;
```

Comment:
Can we please remove lines if not used. Shouldn't have commented out code in samples